ompi/include/mpi.h.in

```diff
@@ -806,10 +791,15 @@ enum {
    protection for this case. */
 
 #if !defined(OMPI_COMPILING_FORTRAN_WRAPPERS)
+/*
+ * Deprecated Defines:
```

Comment:
Can you put the MPI spec version when these were deprecated?

Also, the MPI spec doesn't say whether these are macros or not.  So they're typically not "Deprecated defines".